### PR TITLE
Create backend-tests.yml

### PR DESCRIPTION
**Pull request by tmorris42**
Create a workflow to lint and to run unit tests on the front and backend.

This does NOT run e2e tests on the backend

Fix: #22

### CODE REVIEW

**Review by fyusuf-a — APPROVED** [review]
Awesome!

File: .github/workflows/frontend-tests.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: backend-tests
```
Comment (tmorris42) [inline .github/workflows/frontend-tests.yml:1]:
```suggestion
name: frontend-tests
```

**Comment by tmorris42** [comment]
I see that we have a commit from Linus, which I suspect is a mistake. Maybe we can amend the commit author before merging?

I also made a suggestion for renaming the frontend tests' name.

**Comment by fyusuf-a** [comment]
Well, for Linus, I was having some fun. Sorry, I thought I amended. I am going to make the change and merge.

**Pull request merged**